src/rules/__tests__/prefer-importing-jest-globals.test.ts

```diff
@@ -0,0 +1,54 @@
+import { TSESLint } from '@typescript-eslint/utils';
+import dedent from 'dedent';
+import rule from '../prefer-importing-jest-globals';
+import { espreeParser } from './test-utils';
+
+const ruleTester = new TSESLint.RuleTester({
+  parser: espreeParser,
+  parserOptions: {
+    ecmaVersion: 2015,
+    sourceType: 'module',
+  },
+});
+
+ruleTester.run('prefer-importing-jest-globals', rule, {
```

Comment:
We still need more tests - please think more about the _behaviour_ which isn't necessarily enforced by coverage; e.g. what does this rule do if there are multiple usages of the same function call? what if we're already importing one jest function but not another? what if we reassign a jest function to a variable and then used? what if we import a jest-like function from elsewhere?

In this case the answers to those are pretty obvious from looking over the code, but we want tests to reflect those so that it's obvious if they change

Reply:
This is getting better - not exhaustive but we're at least missing cases for fixing when:
  - there's a hashbang
  - there's a comment at the start of the file (separate from a hashbang) 
  - there's `'use strict'`
  - there's already an import line for `@jest/globals`;

We also should have some more general tests of different complexities and using chains like `describe.each`

Reply:
Added extensive unit tests